qemu/tests/cfg/blockdev_inc_backup_after_commit.cfg

```diff
@@ -40,7 +40,7 @@
     gluster_direct:
         enable_gluster_data = yes
     ceph:
-        enable_ceph_data = yes
```

Comment:
Also let's keep this line in case we did rebase/backing on data, which may needs to specific param setting

Reply:
Have updated it, thanks zhenchao.